docs/en/ingest-management/release-notes/release-notes-8.16.asciidoc

```diff
@@ -46,6 +46,75 @@ impact to your application.
 
+[discrete]
+[[known-issues-8.16.0]]
+=== Known issues
+
+[[known-issue-191661]]
+.{fleet} UI listing shows "No agent found"
+[%collapsible]
+====
+
+*Details*
+
+In the {fleet} UI in {kib}, the listing {agents} might show "No agent found" with a toast message "Error fetching agents" or "Agent policy ... not found".
+
+This error can happen if the {agents} being searched and listed in the UI are using an {agent} policy which doesn't exist.
+
+*Impact* +
+
+As a workaround for the issue, you can upgrade your {stack} to verion 8.16.1. The issue has been resolved by {kibana} link:https://github.com/elastic/kibana/pull/199325[#199325]. 
+
+====
+
+[[known-issue-5952]]
+.{agent} throws exception when uninstalling on Windows
+[%collapsible]
+====
+
+*Details*
+
+{fleet}-managed {agent} sometimes throws an exception when uninstalling on Microsoft Windows systems.
+
+For example:
+
+[source,shell]
+----
+C:\>"C:\Program Files\Elastic\Agent\elastic-agent.exe" uninstall
+Elastic Agent will be uninstalled from your system at C:\Program Files\Elastic\Agent. Do you want to continue? [Y/n]:y
+[====] Attempting to notify Fleet of uninstall  [37s] unexpected fault address 0x18000473ef1
+fatal error: fault
+[signal 0xc0000005 code=0x1 addr=0x18000473ef1 pc=0x9f3004]
+
+goroutine 1 gp=0xc00007c000 m=5 mp=0xc000116008 [running]:
+runtime.throw({0x207a4ba?, 0xa2d986?})
+        runtime/panic.go:1023 +0x65 fp=0xc000067588 sp=0xc000067558 pc=0xcf8c5
+runtime.sigpanic()
+        runtime/signal_windows.go:414 +0xd0 fp=0xc0000675d0 sp=0xc000067588 pc=0xe6a10
+(...)
+        github.com/elastic/elastic-agent/internal/pkg/agent/errors/generators.go:23
+github.com/elastic/elastic-agent/internal/pkg/fleetapi.(*AuditUnenrollCmd).Execute(0xc00073f998, {0x4, 0x23cf148}, 0x0)
+        github.com/elastic/elastic-agent/internal/pkg/fleetapi/audit_unenroll_cmd.go:74 +0x324 fp=0xc000067738 sp=0xc0000675d0 pc=0x9f3004
+runtime: g 1: unexpected return pc for github.com/elastic/elastic-agent/internal/pkg/fleetapi.(*AuditUnenrollCmd).Execute called from 0xc0006817a0
+stack: frame={sp:0xc0000675d0, fp:0xc000067738} stack=[0xc000064000,0xc000068000)
+0x000000c0000674d0:  0x000000c000067508  0x00000000000d14af <runtime.gwrite+0x00000000000000ef>
+0x000000c0000674e0:  0x00000000023c9c90  0x0000000000000001
+0x000000c0000674f0:  0x0000000000000001  0x000000c00006756b
+(...)
+----
+
+For other examples, refer to {agent} link:https://github.com/elastic/elastic-agent/issues/5952#issuecomment-2475044465[issue #5952].
+
+This problem occurs when {agent} notifies {fleet} to audit the uninstall process.
+
+*Impact* +
+
+This issue prevents {agent} from being uninstalled on Windows systems.
+
+We're currently investigating the issue and will update this page when a workaround becomes available.
```

Comment:
Retrying the uninstall again seems like a reasonable work around until we get some evidence it doesn't work, it's not simple to create an agent in this state for me to test.

If that fails, we can post instructions on manually removing the agent service and deleting the files in the installation directory.

Reply:
Sounds good. I've change this to simply "As a workaround, we recommend trying again to uninstall the agent."